turtlebot3c.model

```diff
@@ -2,23 +2,23 @@ type: model
 authority-id: canonical-robotics
```

Comment:
To sign, did you use the key I save in our vault or used a new one?

Reply:
Good question. I probably used the new key...

Reply:
Re-signed with the original key :+1: